File: src/plugin/history/HistoryStack.java
package plugin.history;

import arc.struct.Seq;

public class HistoryStack {
    public Seq<HistoryRecord> stack = new Seq<>();

    public int size() {
        return stack.size;
    }

    public void add(HistoryRecord record) {
        stack.add(record);
    }

    public void removeFirst() {
        stack.remove(0);
    }
}

File: src/plugin/history/HistoryStack.kt
package plugin.history

import arc.struct.Seq

class HistoryStack {
    var stack: Seq<HistoryRecord> = Seq<HistoryRecord>()

    fun size(): Int {
        return stack.size
    }

    fun add(record: HistoryRecord) {
        stack.add(record)
    }

    fun removeFirst() {
        stack.remove(0)
    }
}
